frontend/src/component/admin/banners/BannerModal/BannerModal.tsx

```diff
@@ -61,7 +61,9 @@ export const BannerModal = ({ banner, open, setOpen }: IBannerModalProps) => {
         setLink(banner?.link || '');
         setLinkText(banner?.linkText || '');
         setDialogTitle(banner?.dialogTitle || '');
-        setDialog(banner?.dialog || '');
+        if (typeof banner?.dialog === 'string') {
+            setDialog(banner?.dialog || '');
+        }
```

Comment:
Maybe `setDialog(typeof banner?.dialog === 'string' ? banner?.dialog : '');` instead? Otherwise I think we're not really emptying this value when creating a new banner.